JustSaying.Models/Message.cs

```diff
@@ -3,6 +3,19 @@
 
 namespace JustSaying.Models
 {
+    public class PublishEnvelope
+    {
+        public PublishEnvelope(Message message)
+        {
+            Message = message ?? throw new ArgumentNullException(nameof(message));
+        }
+
+        public Message Message { get; }
+
+        public int? DelaySeconds { get; set; }
+        public IDictionary<string, MessageAttributeValue> MessageAttributes { get; set; }
```

Comment:
Might be nice for the integrator if we initialized this by default so they can just start adding to it immediately, rather than having to initialize it themselves every place it's used.

Reply:
I am in two minds of doing that, since in the majority of cases, it won't be used and creating the dictionary is therefor a wasted allocation.